ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2155,6 +2155,11 @@ router_ospf:
         connected:
           route_map: < route_map_name >
       auto_cost_reference_bandwidth: < bandwidth in mbps >
+      areas:
+        < area >:
+          filter:
+            network: < IPv4 subnet / netmask >
```

Comment:
EOS supports multiple network filters, so we should make this a list `networks`.
Please make the OSPF documentation general for OSPF areas, so we can add more knobs to the same documentation section.